src/librustc/ty/maps/mod.rs

```diff
@@ -254,9 +253,9 @@ define_maps! { <'tcx>
     [] fn item_attrs: ItemAttrs(DefId) -> Lrc<[ast::Attribute]>,
     [] fn trans_fn_attrs: trans_fn_attrs(DefId) -> TransFnAttrs,
     [] fn fn_arg_names: FnArgNames(DefId) -> Vec<ast::Name>,
+    [] fn rendered_const: RenderedConst(DefId) -> String,
```

Comment:
I know this is essentially self explanatory. Still, a short doc comment would be awesome